openml/tasks/task.py

```diff
@@ -36,14 +36,24 @@ class OpenMLTask(OpenMLBase):
 
     Parameters
     ----------
+    task_id : Optional[int]
+        Refers to the unique identifier of OpenML task.
     task_type_id : TaskType
-        Refers to the type of task.
+        Refers to the type of OpenML task.
     task_type : str
-        Refers to the task.
+        Refers to the OpenML task.
     data_set_id: int
         Refers to the data.
     estimation_procedure_id: int
         Refers to the type of estimates used.
+    estimation_procedure_type : str, (optional)
```

Comment:
If optional, I would add the default values to the doc string, e.g.: `str, default=None` following the typing. I think it would also be a good idea to change this for all cases below. It is more helpful than "(optional)" IMO.